src/commands/games/blackjack.ts

```diff
@@ -186,23 +188,46 @@ export class BlackjackCommand extends Command {
   async messageRun(message: Message, args: Args): Promise<Message> {
     // if there are no arguments, then resolve to the default bet amount; if there is only one argument and it is an
     // integer, then this is the bet amount; otherwise, reply that a valid bet amount must be entered
-    const bet = args.finished
-      ? DEFAULT_BET
-      : await args.rest('integer').catch(() => 'please enter a valid bet amount.');
-    if (typeof bet === 'string') return message.reply(bet);
+    const betString = await args.pick('string').catch(() => String(DEFAULT_BET));
 
     const { author, channel } = message;
+    const playerBalance = await getCoinBalanceByUserId(author.id);
+
+    if (isNaN(parseInt(betString, 10))) {
```

Comment:
Can we just `parseInt` once, and then reuse this variable? I know the unhandled `parseInt`s in later parts of this function shouldn't return `NaN`, but it's not a good practice to leave them like that.

Reply:
you've got a point, done